README.md

```diff
@@ -838,6 +839,26 @@ We would rotate our log files with logrotate, by adding the following to `/etc/l
 
 </details>
 
+<a name="multiple"></a>
+## How to save to multiple files?
+<details>
+<summary>Use a separate tool to store all the errors on another log file:</summary>
+
+Install [pino-tee](http://npm.im/pino-tee) with:
+
+```bash
+npm i pino-tee -g
+```
+
+The following writes the log output of `app.js` to `./all-logs`, while
+writing only warnings and errors to `./warn-log:
+
+```bash
+node app.js || pino-tee warn ./warn-logs > ./all-logs
```

Comment:
This should be a single `|`.

![2016-12-13_12-43-51](https://cloud.githubusercontent.com/assets/321201/21151801/d1449cc2-c131-11e6-8185-09cb0681efe9.png)